### ISSUE TITLE
Narrator: Page scan mode skips inputs following labels with absolute position :after pseudo elements

### ISSUE THREAD

**Issue by jayleems2018** [OfficeDev/office-ui-fabric-react#6382]
### Bug Report 
 
 
- __Package version(s)__: (fill this out) 
- __Browser and OS versions__: (fill this out if relevant)
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
 
Requested priority: (Blocking, High, Normal, Low) 
 
Products/sites affected: (if applicable)
 
#### Describe the issue:
 
While use Narrator and Edge with CAPS+M which is to read full screen content, it does not read role like radio button.
 
#### Actual behavior:
 
it does not read role like radio button.

 
#### Expected behavior:
 
it should read role like radio button.

**Comment by micahgodbolt** [comment]
are you specifically talking about the choicegroup? or do you mean all components with roles such as radio button?

**Comment by jayleems2018** [comment]
I mean specifically radio button.

**Comment by micahgodbolt** [comment]
When I use narrator on our choicegroups (radio buttons) i get the following:

> Pick one *, selected, Option A, radio button,

Is this incorrect?

**Comment by jayleems2018** [comment]
Did you follow this step as I mentioned in bug report?
While use Narrator and Edge with CAPS+M which is to read full screen content

If you press tab key, it reads, but our accessibility team requires what if with CAPS +M, which does not read in our case.

**Comment by micahgodbolt** [comment]
Ah, yes. I did not try that. I will.

**Comment by micahgodbolt** [comment]
took me a little while, but it appears to be an issue with the pseudo elements we're using to represent the radio buttons. I don't know why, but when I turn off the CSS for the pseudo elements, it works fine. The markup is perfect.

**Comment by jayleems2018** [comment]
Do you have a sample?

**Comment by micahgodbolt** [comment]
Here is the exact same DOM as the choicegroup spits out, but with none of the styles

https://codepen.io/anon/pen/ZMqoBx

**Comment by micahgodbolt** [comment]
Specifically the display property of the :after pseudo element. https://codepen.io/micahgodbolt/pen/oPaEjL

**Comment by micahgodbolt** [comment]
I'm happy to call this a narrator bug.

**Comment by micahgodbolt** [comment]
I've logged this bug https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/19037570/

**Comment by aneeshack4** [comment]
Closing as this has been logged with Edge
